resolver.js

```diff
@@ -33,45 +34,68 @@ var domainForResolver = function domainForResolver(req) {
 }
 
 var defaultResolver = function resolver(host, url, req) {
+    var wikiDomain = domainForResolver(req)
+    if (wikiDomain === null) {
+        return Promise.resolve(null)
+    }
+
+    return fetchWikiForDomain(wikiDomain)
+        .then(function (response) {
+            if (!response || !response.data) {
+                return null
+            }
+
+            let backend = response.data.wiki_queryservice_namespace.backend
```

Comment:
We could also cache this computation. I steered away from it as it would mean even more noise in this changeset. Any opinions?